src/components/common/TopDrops/index.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```

Comment:
We are gonna probably update this component at the end of the project with actual animations maybe on canvas or SVG but for now it's ok.

Reply:
correct, the idea is replace the images by a animate canvas

Reply:
Yes, but for now we are gonna leave as it is.